docs/data/sdks/android-ampli.md

```diff
@@ -7,7 +7,7 @@ icon: material/android
 !!!note
     This page covers the Android Java and Kotlin runtimes. All (Itly) runtimes are deprecated. If you are still using an (Itly) runtime, see the **[migration guide](#migrating-from-previous-version)** to upgrade to the newest runtime. Docs for the Itly version are available **[here](browser)**.
 
-Amplitude Data supports tracking analytics events from Android apps (API 22 and above) written in Kotlin and Java.
+Amplitude Data supports tracking analytics events from Android apps written in Kotlin and Java.
```

Comment:
Lgtm. Nit: do you think we should still maintain the API information here? Or they can just refer from core SDK info?

Reply:
I think that they can refer to the core SDK for minSdkVersion so that we don't need to maintain in two places.